server/service/src/processors/transfer/requisition/test.rs

```diff
@@ -217,7 +217,7 @@ impl RequisitionTransferTester {
             r.store_id = request_store.id.clone();
             r.r#type = RequisitionRowType::Request;
             r.status = RequisitionRowStatus::Draft;
-            r.created_datetime = NaiveDate::from_ymd_opt(2021, 01, 01)
+            r.created_datetime = NaiveDate::from_ymd_opt(2021, 1, 1)
```

Comment:
For some reason I prefer the previous version, but again I think it's fine to change.